bin/create_yml.pl

```diff
@@ -203,9 +219,12 @@
     }

```

Comment:
What about adding a STDERR message in the if statement here. To give us a tiny chance to see this if ever triggered

Reply:
it really should be triggered everytime, since the regex really does not look for anything being added. Maybe I should just remove this. The funciton this was to solve has been solved elsewhere by using flags in loqusdb processes instead

Reply:
OK removing sounds even better!